addons/web_editor/static/src/js/editor/odoo-editor/src/style.css

```diff
@@ -26,11 +79,8 @@
 .odoo-editor-editable.o_resizing_column ::selection, .odoo-editor-editable.o_resizing_row ::selection {
     background-color: transparent;
 }
-
-.oe-floating {
-    box-shadow: 0px 3px 18px rgba(0, 0, 0, .23);
-    border-radius: 4px;
-    position: absolute;
```

Comment:
Why remove this ?

Reply:
Uh I didn't, I just added things before and after it so git is confused.